docs/section11/Rule11-15.md

```diff
@@ -0,0 +1,67 @@
+# ServiceWaterHeating - Rule 11-15
+
+**Schema Version:** 0.0.37  
+**Mandatory Rule:** True  
+**Rule ID:** 11-15  
+
+**Rule Description:** "Service water loads and use shall be the same for both the proposed design and baseline building design and shall be documented by the calculation procedures described in Section 7.4.1.  
+Exceptions:  
+(1) Energy Efficiency Measures approved by the Authority Having Jurisdiction are used in the proposed model
+(2) SWH energy consumption can be demonstrated to be reduced by reducing the required temperature of service mixed water, by increasing the temperature, or by increasing the temperature of the entering makeup water. "  
+
+**Rule Assertion:** Options are PASS/FAIL/NOT_APPLICABLE/UNDETERMINED  
+**Appendix G Section Reference:** Table G3.1 #11, baseline column, (g)  
+
+**Evaluation Context:** Each SWH use  
+**Data Lookup:**   
+**Function Call:**
+- **get_SWH_uses_associated_with_each_building_segment**  
+- **ge_component_by_id**  
+
+**Applicability Checks:**
+- create a list of swh use ids that need to be checked - we'll create the list from both the proposed and baseline models: `swh_use_ids = []`
+- look at each building in the B_RMD: `for building in B_RMD:`
+    - create a boolean is_applicable and set it to false: `is_applicable = false`
+    - look at each building segment in the building: `for building_segment in building.building_segments:`
+    - get the service water heating uses in the building segment `service_water_heating_use_ids = get_SWH_uses_associated_with_each_building_segment(P_RMD, building_segment.id)`
+    - look at each service water heating use id: `for swh_use_id in service_water_heating_use_ids:`
+        - append the swh_use_id to the list of swh use ids: `swh_use_ids.append(swh_use_id)`
```

Comment:
Not following why this step is needed, looks like you can just use service_water_heating_use_ids

Reply:
Maybe this is missing an if statement?